doc/source/development/community.rst

```diff
@@ -100,6 +100,8 @@ The pandas mailing list `pandas-dev@python.org <mailto://pandas-dev@python
 conversations and to engage people in the wider community who might not
 be active on the issue tracker but we would like to include in discussions.
 
+Join the mailing list and view the archives `here <https://groups.google.com/forum/?fromgroups#!forum/pydata>`_.
```

Comment:
This link seems to be for the pydata group rather than the pandas-dev mailing list.
I think this is the correct link: https://mail.python.org/mailman/listinfo/pandas-dev

I find it weird that the README links to this as the 'mailing list', rather than 'pydata group' or 'pydata mailing list', @mroeschke should that be changed?

Reply:
@Aloqeely I can update the link here and in the README

Reply:
Yeah I think @Aloqeely's link would be more appropriate

I think the link in the README is fine; that mailing list is more informational and is designed to have less technical discussion